sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerConfiguration.java

```diff
@@ -82,17 +83,12 @@ public JwtDecoder jwtDecoder() {
      * bean to override it.
      */
     @Configuration
-    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
     @EnableWebSecurity
-    public static class DefaultAzureOAuth2ResourceServerWebSecurityConfigurerAdapter extends
-        WebSecurityConfigurerAdapter {
-
+    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
+    public static class DefaultWebApiSecurityConfigurerAdapter extends AADResourceServerSecurityConfigurerAdapter {
```

Comment:
How about rename like this:
1. `DefaultWebApiSecurityConfigurerAdapter ` -> `DefaultAADResourceServerWebSecurityConfigurerAdapter`.
2. `DefaultAADWebSecurityConfigurerAdapter` to `DefaultAADWebAppliationWebSecurityConfigurerAdapter`.
3. `AADWebSecurityConfigurerAdapter` -> `AADWebApplicationWebSecurityConfigurerAdapter`.
4. `AADResourceServerSecurityConfigurerAdapter` -> `AADResourceServerWebSecurityConfigurerAdapter `

Reply:
But since we've released the AAD starter, now changing the name of `AADWebSecurityConfigurerAdapter ` will cause a breaking change.

Reply:
Then just rename resource server related name.